tutorials/cloudsdk-integrate-cap/cloudsdk-integrate-cap.md

```diff
@@ -11,7 +11,7 @@ author_profile: https://github.com/MatKuhr
 ## Prerequisites
  - SAP Cloud Application Programming Model
  - Basic knowledge of Spring Boot and Java (optional)
- - An account on [cloud foundry](https://developers.sap.com/group.scp-1-get-ready.html)
+ - An account on [Cloud Foundry](group.scp-1-get-ready)
```

Comment:
Even when removing the `.html` it doesn't work. Could you please create a suggestion what would be the correct format? E.g.

```suggestion
 - An account on [Cloud Foundry](group.scp-1-get-readyyyyy)
```